### PR TITLE
Create ui-c8y-1021-0-0-us-locales-support.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by janhommes

Ticket: [MTM-59798](https://cumulocity.atlassian.net/browse/MTM-59798)
Original PR: [6774](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/6774)

[MTM-59798]: https://cumulocity.atlassian.net/browse/MTM-59798?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-us-locales-support.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: us locales support
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-us-locales-support.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Support for US locales
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-us-locales-support.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59798
+version: 1021.0.0
+---
+us locales support
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-us-locales-support.md:17]: The Claude AI suggested to use the following body:
```suggestion
To better serve our users in the United States, support for US locales has been added. With this change, the application now recognizes and correctly handles locale-specific settings such as date and time formats, number formats, and currency symbols for the United States. This improves the user experience for US-based users by ensuring the application displays information in a way that is familiar and expected for their locale.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2303

**Pull request closed without merging**